### PR TITLE
LFS: part 11, structs

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

**Review by ikbenlike — APPROVED** [review]
LGTM

**Review by Dennisbonke — APPROVED** [review]
LGTM

File: options/glibc/include/sys/user.h [hunk 1]
```diff
@@ -42,4 +42,8 @@ struct user {
 }
 #endif
 
+#define PAGE_SHIFT 12
```
Comment (Geertiebear) [inline options/glibc/include/sys/user.h:45]: What is this and is it platform dependent?

Comment (no92) [reply options/glibc/include/sys/user.h:45]: It's apparently a define for gdb, and it is not platform-dependent (in glibc + musl). It does not reflect true page size, as you should use the function for that.

**Pull request merged**